vsintegration/src/FSharp.Editor/LanguageService/LanguageService.fs

```diff
@@ -316,12 +317,28 @@ and
     override this.Initialize() =
         base.Initialize()
 
+        let workspaceChanged (args:WorkspaceChangeEventArgs) =
+            match args.Kind with
+            | WorkspaceChangeKind.ProjectAdded
+            | WorkspaceChangeKind.ProjectChanged
```

Comment:
@Pilchie I have had it both ways.  I will go back to per project ... it seems like less overhead.